rllib/connectors/input_output_types.py

```diff
@@ -1,75 +0,0 @@
-from enum import Enum
```

Comment:
This is no longer needed.
We now provide some light-weight utility methods within the ConnectorV2 API to simplify batch building, if desired by the users. However, they can also always just manually manipulate the batch (it's always just a dict), if they want to.